src/components/content/content-blocks/block-quote/block-quote.tsx

```diff
@@ -7,9 +7,10 @@ import {
 import { st, classes } from "./block-quote.st.css";
 import { mlUtils } from "../../../../lib/ml-utils";
 
+
 export const BlockQuote = ({
 	componentData,
-	className,
+	className
```

Comment:
Seems like unnecessary changes to this file.

Reply:
leftovers from an attempted refactoring. If the change turns out to be detrimental to the project, we can roll it back in the next ML Hackathon.